### PR TITLE
2023q3: add Cirrus-CI report

### PR DESCRIPTION
**Pull request by emaste**

### CODE REVIEW

File: website/content/en/status/report-2023-04-2023-06/cirrus.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+=== Cirrus-CI
+
+Links: +
+link:https://cirrus-ci.com/github/freebsd/[FreeBSD Cirrus-CI Repositories] URL: https://cirrus-ci.com/github/freebsd/
+link:https://cirrus-ci.com/github/freebsd/freebsd-src[FreeBSD src CI] URL: https://cirrus-ci.com/github/freebsd/freebsd-src
+link:https://cirrus-ci.com/github/freebsd/freebsd-doc[FreeBSD doc CI] URL: https://cirrus-ci.com/github/freebsd/freebsd-doc
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/cirrus.adoc:6]:
```suggestion
link:https://cirrus-ci.com/github/freebsd/[FreeBSD Cirrus-CI Repositories] URL: link:https://cirrus-ci.com/github/freebsd/[]
link:https://cirrus-ci.com/github/freebsd/freebsd-src[FreeBSD src CI] URL: link:https://cirrus-ci.com/github/freebsd/freebsd-src[]
link:https://cirrus-ci.com/github/freebsd/freebsd-doc[FreeBSD doc CI] URL: link:https://cirrus-ci.com/github/freebsd/freebsd-doc[]
```

File: website/content/en/status/report-2023-04-2023-06/cirrus.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Contact: Brooks Davis <brooks@FreeBSD.org>
+Contact: Ed Maste <emaste@FreeBSD.org>
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+Cirrus-CI is a hosted continuous integration service that supports open source projects with CI servies on Linux, Windows, macOS, and FreeBSD.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/cirrus.adoc:12]:
```suggestion
Cirrus-CI is a hosted continuous integration service that supports open source projects with CI services on Linux, Windows, macOS, and FreeBSD.
```

**Review by lsalvadore — APPROVED** [review]
Approved once the small nits I have found are fixed.

**Comment by emaste** [comment]
Shall I just push to main?

**Comment by emaste** [comment]
Merged to main

**Pull request closed without merging**